src/deploy/NVA_build/noobaa_log_segregate.sh

```diff
@@ -49,28 +57,43 @@ oldest_log_file() {
     echo "$earliest_filename"
 }
 
+move_old_small_bucket_logs() {
```

Comment:
Does this loop run continously ?

Reply:
NO. It will check if there are files in /var/log/noobaa_bucket_logs/ with .log extension. If it is then it will move it. If not then it will come out of loop.

Reply:
okay